boot/modeenv.go

```diff
@@ -44,7 +44,25 @@ type Modeenv struct {
 	read bool
 }
 
+var readModeenv = readModeenvImpl
+
+// ReadModeenv attempts to read the file specified by
+// "rootdir/dirs.SnapModeenvFile" as a Modeenv.
```

Comment:
the "" make it seem those are literal paths,  I prefer simply

```
... file at <rootdir>/var/lib/snapd/modeenv as a Modeenv.
```

@bboozzoo this relates also to my comment about the Join in the Impl

Reply:
This is fixed now.